apps/Persepolis Download Manager/install

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd $HOME
+wget -O adwaita-qt.deb http://mirror.datamossa.io/raspbian/pool/main/a/adwaita-qt/adwaita-qt_1.0-2_armhf.deb
+wget -O youtube-dl.deb http://ftp.us.debian.org/debian/pool/main/y/youtube-dl/youtube-dl_2021.06.06-1_all.deb
```

Comment:
I think it's better to install `youtube-dl` from `pip3`.

Reply:
This is the route YouTubuddy takes.
https://github.com/Botspot/youtubuddy/blob/main/gui#L43

Reply:
> I think it's better to install `youtube-dl` from `pip3`. 


Why? The deb downloaded in the script is latest version of youtube-dl (2021.6.6), same with the [pip one](https://pypi.org/project/youtube_dl/). 

I prefer using deb one because it is `purge-installed`able, and I think using deb (apt) in pi-apps is more reliable than pip since there are more diagnose of apt in pi-apps but not pip. 

Also, if we separate to use pkg-install and pip to install dependencies, that will definetely extend the time to install the app.